README.md

```diff
@@ -24,6 +24,18 @@ This subgraph indexes events emitted by the "Optimistic Oracle" contracts. The c
 - (unofficial) Boba: https://thegraph.com/hosted-service/subgraph/md0x/boba-oo-staging
 - (staging) Goerli: https://thegraph.com/hosted-service/subgraph/md0x/goerli-oo-staging
 
+## Optimistic Oracle V2 Events and Calls:
+
+This subgraph indexes events and function calls by the "Optimistic Oracle V2" contracts. The code can be found in `packages/optimistic-oracle-v2`
+
+- (unofficial) Mainnet: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/mainnet-oo-v2-staging
+- (unofficial) Polygon: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/polygon-oo-v2-staging
+- (unofficial) Arbitrum: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/arbitrum-oo-v2-staging
+- (unofficial) Optimism: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/optimism-oo-v2-staging
+- (unofficial) Boba: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/boba-oo-v2-staging
+- (staging) Goerli: https://thegraph.com/hosted-service/subgraph/ryanwolhuter/goerli-oo-v2-staging
```

Comment:
Are they deployed?
The links doesn't work

Reply:
whoops I made a mistake in the urls lol, please take another look. they are all deployed